readme.md

```diff
@@ -42,40 +48,40 @@ console.log(await getStream(stream));
 
 ## API
 
-The methods returns a promise that resolves when the `end` event fires on the stream, indicating that there is no more data to be read. The stream is switched to flowing mode.
```

Comment:
This description is specific to Node.js streams (e.g. the `end` event), which won't work with the upcoming PR I have to add support for web streams.